tests/fixtures/testrepo/upstream/mock/el7.cfg

```diff
@@ -0,0 +1,49 @@
+config_opts['root'] = 'foreman-scl-7-x86_64'
```

Comment:
given we're not actually running mock in the tests, would an empty file suffice? or one with a single comment "here shall be a config"?

Reply:
Sure.